.github/workflows/sandbox.yaml

```diff
@@ -0,0 +1,28 @@
+name: Test Getting started
```

Comment:
Can we  add the docs to test this in the contributing guide in this PR. Its been long pending one